roles/kubernetes-apps/ansible/templates/dashboard.yml.j2

```diff
@@ -127,7 +140,7 @@ spec:
       labels:
         k8s-app: kubernetes-dashboard
     spec:
-{% if not dashboard_use_custom_certs %}
+{% if dashboard_use_custom_certs %}
```

Comment:
1.8 doesn't need the initContainer but we still need the volume mount. Please take another look at https://github.com/kubernetes/dashboard/blob/master/src/deploy/recommended/kubernetes-dashboard.yaml#L114-L146

Reply:
ok  remove  it.